### ISSUE TITLE
[chandler] feature: Ability to reorganize contact types

### ISSUE THREAD

**Issue by phorcys420** [monicahq/monica#6778]
![image](https://github.com/monicahq/monica/assets/57866459/42ea884b-7e4e-414f-baed-09aed7841766)

---

I would like to be able to define which contact types are at the top and which ones are at the bottom.
You are technically already able to do this by deleting everything and recreating them or modifying all of them one by one based on their position in the list but it's clearly not optimal.

I think there should be something like this in the "Manage contact information types" page.
![image](https://github.com/monicahq/monica/assets/57866459/38d8d2f2-7b37-4b27-83f4-0db3a49bed3a)

**Comment by SecurID** [comment]
Not sure if that is a good solution. The contact types are a list of 9 items. It's not that you would have to search for minutes to find the correct one.

**Comment by phorcys420** [comment]
> It's not that you would have to search for minutes to find the correct one.

No, but it's still a good UX improvement to not have to go all the way down the dropdown, I would have the most used ones at the top.
I do this quite a lot so that's why it'd be useful.

**Comment by phorcys420** [comment]
Also I'd like to mention that it doesn't show in the same order in the edit page.

![image](https://github.com/monicahq/monica/assets/57866459/c1c4c480-1c72-40c4-8c90-92b7c3c46ed2)
